src/check.php

```diff
@@ -58,7 +58,7 @@ function loghistory($link,$userid,$ip,$ua,$outcome)
         $sql="INSERT INTO `blockip` VALUES (?,CURRENT_TIMESTAMP)";
         $res=sqlexec($sql,array(getUserIP()),$link);
     }
-    die("1");
+    die("0");
```

Comment:
thanks ;) there could still be a timing issue. But for now that's overkill to fix.